src/test/regress/sql/pg16.sql

```diff
@@ -591,28 +591,8 @@ REVOKE role1 FROM role2;
 RESET citus.log_remote_commands;
 RESET citus.grep_remote_commands;
```

Comment:
```suggestion
```
Let's remove these lines such that we see the propagated "GRANT" log remote commands